java/vector/pom.xml

```diff
@@ -44,17 +49,16 @@
       <groupId>com.fasterxml.jackson.core</groupId>
       <artifactId>jackson-databind</artifactId>
     </dependency>
+    <dependency>
+      <groupId>com.carrotsearch</groupId>
+      <artifactId>hppc</artifactId>
+      <version>0.7.2</version>
+    </dependency>
```

Comment:
why is this needed ?

Reply:
This was in the commit history and I wasn't sure if it should be removed, i have removed it here https://github.com/sirensolutions/arrow/pull/16/commits/5d79a1902316072bb4b05250736b1fe669182141